modules/upload_contacts_from_file.py

```diff
@@ -0,0 +1,121 @@
+import csv
+import re
+from modules import utils, visit_date_helper
```

Comment:
It might be better to

```
 from modules.utils import date_string_ymd_to_date, date_string_mdy_to_date, get_current_time
from modules.visit_date_helper import add_or_subtract_days
```

and then not prefix `utils.` and `visit_date_helper.` in the functions as they are called. This is a bit more explicit about what is happening and a bit less verbose (in the function invocation).